### DISCUSSION TITLE
Your Discord link doesn't seem to work!

### DISCUSSION THREAD

**Discussion by oywino** [music-assistant/support#1411] [Q&A] [answered]
Sorry if this is not the right place to ask for help, but my attempts to visit your Discord server fails.
This is your link: [https://discord.com/channels/753947050995089438/753947050995089442](url)
But Discord persistently complains that I have reached a server that has no textual content.

I need an invite link, please?

**Comment by OzGav** [comment]
Try this https://discord.com/channels/753947050995089438/1083284385974980728

**Reply by oywino** [reply]
Nope - still no-Go.  This is what I get:
![image](https://github.com/music-assistant/hass-music-assistant/assets/18447119/fe2a9566-f387-4cdc-a5e7-5a3cce1fa449)

**Reply by OzGav** [reply]
Do you have a discord account?

**Reply by oywino** [reply]
Yes, of course. I use Discord on a daily basis.

**Reply by OzGav** [reply, marked as answer]
Don’t know why it isn’t working as it works ok for me. Try this one https://discord.gg/j2vSKS7J

**Reply by oywino** [reply]
That worked 👍  Many thanks 🙂
